src/ir/auth_logic/souffle_emitter_test.cc

```diff
@@ -42,219 +42,32 @@ Program BuildPredicateTestProgram() {
                                                      datalog::kPositive))))));
 }
 
-TEST(EmitterTestSuite, SimpleTest) {
```

Comment:
Removed most of the tests written earlier as they would fail with current changes. Before this PR, relation declarations were generated from predicate name(As a word around I guess). For this reason, tests written earlier did not have correct relation declarations and were missing type declarations, canActAs declaration etc. I wrote new tests with couple of base cases (empty auth logic and simple predicate fact). More tests needs to be added moving forward. I will take care of that in another PR.

Reply:
I see. Sounds good.